WcaOnRails/app/assets/stylesheets/users.scss

```diff
@@ -42,3 +42,7 @@ input[type="radio"].center-vertically {
 #avatar-img {
   max-height: 400px;
 }
+
+#email .help-block {
```

Comment:
If my suggestion on `edit.html.erb` works, then I think we can get rid of this css.